src/api/app/components/workflow_artifacts_per_step_component.rb

```diff
@@ -34,6 +37,37 @@ def call
 
   private
 
+  def artifacts_for_set_flag(parsed_artifacts)
+    capture do
+      list_of_flags(parsed_artifacts[:flags])
+    end
+  end
+
+  def list_of_flags(flags)
+    flags.each do |flag|
+      concat(flag_step_sentence(flag))
+    end
+  end
+
+  def flag_step_sentence(flag)
+    if flag[:package]
+      path = helpers.package_show_path(project: flag[:project], package: flag[:package])
```

Comment:
:thought_balloon: The only thing I would change (but it's a matter of personal style maybe) is that extract a function to get the `path` and another to get the `text`. That way, `flag_step_sentence` will be smaller and simpler.

Reply:
Yeah will do that. Codeclimate is also complaining about that